com.unity.render-pipelines.universal/Editor/EditorUtils.cs

```diff
@@ -25,17 +26,19 @@ static class EditorUtils
         public const int lwrpAssetCreateMenuPriorityGroup2 = CoreUtils.assetCreateMenuPriority1 + 50;
         public const int lwrpAssetCreateMenuPriorityGroup3 = lwrpAssetCreateMenuPriorityGroup2 + 50;
 
+        public enum Unit { Metric, Percent }
```

Comment:
double check if this should be public, otherwise you need to add public API docs.

Reply:
The name of this Enum is not very descriptive as well, even for internal code. We should make a better descriptive name.

Reply:
Made it internal, and that seems to be working fine.
Naming is from HDRP and will make it easier to adopt their way of using it when we go over to volumes